p/themes/Origine/origine.css

```diff
@@ -175,7 +175,6 @@ form th {
 	border-radius: 3px;
 	min-height: 37px;
```

Comment:
No need for this line either.

Reply:
did not check the min-height. Will do it. Thanks for that hint.

Reply:
it is copied from base-theme/base.css